jfr-streaming/src/g1GcTest/java/io/opentelemetry/contrib/jfr/metrics/G1MemoryLimitMetricTest.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics;
+
+import org.junit.jupiter.api.Test;
+
+class G1MemoryLimitMetricTest extends AbstractMetricsTest {
```

Comment:
Should we group all the memory related tests together?

Reply:
Ok I grouped the memory related tests into classes per their GC